test/telemetry.spec.ts

```diff
@@ -1,19 +1,48 @@
-/* eslint-disable class-methods-use-this, @typescript-eslint/no-unused-vars */
+/* eslint-disable class-methods-use-this, max-classes-per-file, no-underscore-dangle, @typescript-eslint/no-unused-vars */
```

Comment:
Would it be a good idea to test error handling too? I'm not sure either way; just wanted to get your thoughts.

Reply:
Did you have some examples in mind? There are some tests here which verify behaviors when `sendSpan` or `sendCount` throw errors.

Reply:
Ah ok, I just missed those.